src/views/product-tutorials-view/components/sitemap/index.tsx

```diff
@@ -0,0 +1,33 @@
+import Link from 'next/link'
+import {
+  getCollectionSlug,
+  getTutorialSlug,
+} from 'views/collection-view/helpers'
+import { ProductPageData } from 'views/product-tutorials-view/server'
+
+export function ProductTutorialsSitemap({
+  collections,
+}: {
+  collections: ProductPageData['pageData']['collections']
+}): React.ReactElement {
+  return (
+    <ul>
```

Comment:
Stub version of this component. Wanted to separate it out to keep the view file small